homework/content.js

```diff
@@ -1,37 +1,44 @@
 'use strict';
 
 (function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
 
-  function start() {
-    window.addEventListener('note-open', function(event) {
-      var note = event.detail;
-      resetWrapper();
-      drawNote(note);
-    });
-  }
+    var Content =function() {
+        this._wrapper = document.querySelector('#note-content-wrapper');
+    }
+    
+    var ContentNote = new Content();
+  
+    Content.prototype.start = function() {   //function start()
+        window.addEventListener('note-open', function(event) {
```

Comment:
You should bind the `listener` argument with `this`, rather than an anonymous function like this.